examples/team_simple.yml

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: network-test
+  vars:
+    network_connections:
+      # Specify the team profile
+      - name: team0
+        state: up
+        type: team
+        interface_name: team0
+        # ip configuration (optional)
+        ip:
+          address:
+            - "192.0.2.24/24"
+            - "2001:db8::23/64"
+
+      # add an team profile to the team
+      - name: member1
+        state: up
+        type: team
+        interface_name: team1
+        master: team0
+
+      # add a second team profile to the team
+      - name: member2
+        state: up
+        type: team
+        interface_name: team2
+        master: team0
```

Comment:
I think this is fine but maybe it would be better if the port interfaces are other types like ethernet.. What do you think @tyll?

Reply:
yes, ethernet would be better. Just wondering, does it work to create team on top of team?